axon-5/api-changes.md

```diff
@@ -447,13 +448,13 @@ Here's an example of how to register a `DefaultCommandGateway` through the `regi
 ```java
 public static void main(String[] args) {
     MessagingConfigurer.create()
-                       .componentRegistry(registry -> registry.registerComponent(
-                               CommandGateway.class,
-                               config -> new DefaultCommandGateway(
-                                       config.getComponent(CommandBus.class),
-                                       config.getComponent(MessageTypeResolver.class)
-                               )
```

Comment:
I don't think this change is correct to be honest. Did Intellij do this? For me, it keeps the indentation of code snippets as is.